caffe2/utils/threadpool/pthreadpool_impl.cc

```diff
@@ -28,7 +28,20 @@ void pthreadpool_compute_1d(
 }
 
 size_t pthreadpool_get_threads_count(pthreadpool_t threadpool) {
-  return reinterpret_cast<caffe2::ThreadPool*>(threadpool)->getNumThreads();
```

Comment:
Just as a point of style, I prefer to write

```
if (unusual_condition) {
  return stub_value;
}
do_real_work;
return real_value;
```

as opposed to 

```
if (typical_condition) {
  do_real_work;
  return real_value;
}
return stub_value;
```

I find that it improves readability, especially as "do_real_work" gets long.

Not necessary to change.  Just a suggestion.

Reply:
Sure. It does improve readability in longer code.